src/layout/Columns/index.js

```diff
@@ -16,26 +17,33 @@ const Left = styled.div`
     padding: ${({theme}) => theme.spacing.L};
     width: 100%;
     background: ${({theme}) => theme.color.white};
-    ${SCREEN.DESKTOP_AND_TABLET} {
+    ${SCREEN.DESKTOP} {
         padding: ${({theme}) => `${theme.spacing.XXXL} ${theme.spacing.M} ${theme.spacing.XXXL} ${theme.spacing.M}`};
         width: 55%;
     }
 `
 
 const Right = styled.div`
     display: flex; 
+    flex-direction: column;
     justify-content: center;
     background: ${({theme}) => theme.color.grey};
     width: 100%;
     padding: ${({theme}) => theme.spacing.M};
-    ${SCREEN.DESKTOP_AND_TABLET} {
+    ${SCREEN.DESKTOP} {
         width: 45%;
         padding: 7rem;
     }
+
+    .cnil {
```

Comment:
it could be better to move it in Home page to keep the layout the most pure as possible

Reply:
done